docs/docs/concepts/structure/instance.mdx

```diff
@@ -2,8 +2,23 @@
 title: Instance
 ---
 
-import InstanceDescription from './\_instance_description.mdx';
+## Instance Structure
 
-<InstanceDescription name="InstanceDescription" />
+An instance is the top node in ZITADEL's data hierarchy.
+Within an instance all the default [settings](/docs/concepts/structure/policies),
+such as branding, login policy, password policy, etc. for the system can be configured.
+One instance normally runs on one domain and has one issuer (e.g login.customer.com).
```

Comment:
```suggestion
One instance normally runs on one domain and represents one issuer (e.g login.customer.com).
```

Reply:
ZITADEL is able to respond on multiple domains, but only one can be the main domain which is used to send mails and other things like the console access.

Reply:
I think it is worth mentioning what I stated above

Reply:
😄 ok

Reply:
I thought so 😁